troubadix/reporter.py

```diff
@@ -184,49 +187,79 @@ def report_statistic(self) -> None:
         if not self._statistic:
             return
 
-        if self._fix:
-            self._term.print(
-                f"{'Plugin':48} {'  Errors':8} {'Warnings':8} {'   Fixes':8}"
-            )
+        if self._fix and self._ignore_warnings:
+            line = f"{'Plugin':48} {'  Errors':8} {'   Fixes':8}"
```

Comment:
```suggestion
            header = f"{'Plugin':48} {'  Errors':8} {'   Fixes':8}"
```

Rename the variable maybe?

Reply:
Nah. anyways.